File: TV Discount.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int a,b,c,d;
        scanf("%d%d%d%d",&a,&b,&c,&d);
        if(a-c>b-d)
        printf("Second\n");
        else if(a-c<b-d)
        printf("First\n");
        else
        printf("Any\n");
    }
}

File: TV Discount.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int a=sc.nextInt();
            int b=sc.nextInt();
            int c=sc.nextInt();
            int d=sc.nextInt();
            if(a-c<b-d)
            System.out.println("First");
            else if(a-c>b-d)
            System.out.println("Second");
            else
            System.out.println("Any");
        }
    }
}